lib/gui/app/modules/image-writer.js

```diff
@@ -296,3 +304,28 @@ exports.flash = (image, drives) => {
     windowProgress.clear()
   })
 }
+
+/**
+ * @summary Cancel write operation
+ * @function
+ * @public
+ *
+ * @example
+ * imageWriter.cancel()
+ */
+exports.cancel = () => {
+  analytics.logEvent('Cancel', {
```

Comment:
If a user is flashing multiple drives, I guess they might cancel after some drives have completed but before all drives have completed - I wonder if that'd be useful analytics-data to log?

Reply:
Because of how multi-writes works right now, drives basically finish at the same time. In the future that might change so we'll have to worry about that then though.